src/styles/blocks/section.scss

```diff
@@ -0,0 +1,142 @@
+@use "../utils/mixins" as *;
+
+.section {
+
```

Comment:
remove redundant white spaces here and below

Reply:
all wishes achieve